src/featurization_utils/src/texture_utils.py

```diff
@@ -53,6 +53,25 @@ def measure_3D_texture(
     -------
     dict
         A dictionary containing the object ID, texture name, and texture value.
+        keys (feature names):
+            - object_id
+            - texture_name
+            - texture_value
+
+        Texture names and values:
+            - Angular.Second.Moment
+            - Contrast
+            - Correlation
+            - Variance
+            - Inverse.Difference.Moment
+            - Sum.Average
+            - Sum.Variance
+            - Sum.Entropy
+            - Entropy
+            - Difference.Variance
+            - Difference.Entropy
+            - Information.Measure.of.Correlation.1
+            - Information.Measure.of.Correlation.2
```

Comment:
Reading the naming patterns here I wondered: could these be renamed to the following in order to simplify (all are measures)? I also wondered about what `1` and `2` represent; is there a more specific name we could provide to help indicate by name how they differ? 
```suggestion
            - Information.Correlation.1
            - Information.Correlation.2
```